src/sonic-bgpcfgd/setup.py

```diff
@@ -9,7 +9,7 @@
       author_email='pavelsh@microsoft.com',
       url='https://github.com/Azure/sonic-buildimage',
       packages=setuptools.find_packages(),
-      scripts=['bgpcfgd'],
+      scripts=['bgpcfgd','bgpmon.py'],
```

Comment:
can you do console_script packaging? it is better to do that and we can get rid of .py suffix.

https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html

Reply:
@lguohan Agreed.  I have modified to use console_scripts packaging and it indeed got rid of the .py suffix.
Code re-tested to be functioning as before.